purchase_procurement_analytic/models/stock.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class StockMove(models.Model):
+    _inherit = 'stock.move'
+
+    @api.model
+    def _prepare_procurement_from_move(self):
+        res = super(StockMove, self)._prepare_procurement_from_move()
+        res['account_analytic_id'] = self.procurement_id.account_analytic_id.id
```

Comment:
@pedrobaeza 

this piece of code was in original module and It's not necessary.

I remove it.